README.md

```diff
@@ -18,17 +18,17 @@ Prebuilt binaries can be found [in releases](https://github.com/JetBrains/skia-p
 Note: Better check build.yml for the detailed command for your machine
 
 ```sh
-python3 script/checkout.py --version m138-9e6b5bff162
+python3 script/checkout.py --version m138-c1caa59365c
```

Comment:
We don't need to update README for each commit. Let's use some placeholder instead?